community/contributing/content_guidelines.rst

```diff
@@ -0,0 +1,95 @@
+.. _doc_content_guidelines:
+
+Content guidelines
+==================
+
+This document is here to help us assess what we should include in the official
+documentation. Below, you will find a couple of principles and recommendations
+to write accessible content.
+
+We want to achieve two goals:
+
+1. **Empathize with our users**. We should write in a way that makes it easy for
+   them to learn from the docs.
+2. **Write a complete reference manual**. Our goal here is not to teach
+   programming foundations. Instead, we should provide a reference for how
+   Godot’s features work.
+
+Guidelines and principles
+-------------------------
+
+Below are the guidelines we should strive to follow. They are not hard rules,
+though: exceptionally, a topic will require breaking one or more of these.
+Still, we should strive to achieve the two goals listed above.
+
+Writing complete and accessible documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**A feature doesn’t exist unless it is documented**. If a user can’t find
+information about a feature and how it works, it doesn’t exist to them. We
+should ensure that we cover everything Godot does.
+
+.. note::
+
+   When adding or updating an engine feature, the documentation team needs to
+   know about it. Contributors should open an issue on the godot-docs repository
+   when their work gets merged and requires documentation.
+
+Do your best to keep documents **under 1000 words in length**. If a page goes
+past that threshold, consider splitting it into two parts, if possible. Limiting
+page size forces us to write concisely and to break large documents so they each
+focus on a particular problem.
+
+Make it clear what **problem** each page or section of a page tackles and what
+the user will learn from it. Users need to know if they’re reading the correct
+guide to solving problems they encounter. For example, instead of writing the
+heading “Signals”, consider writing “Reacting to changes in another object with
```

Comment:
Similarly to the use of the typographical single quote, I don't think we should use typographical double quotes in the rst source.

We currently do only in a few pages, probably from your own contributions :)
```
$ rg "“"
tutorials/export/exporting_pcks.rst
58:and go to Project/Export and click on “Export PCK/Zip”. Also make sure to have
94:validation code as a layer on top. The following example expects a “mod.pck”
96:“mod_scene.tscn” test scene in its root.

tutorials/rendering/viewports.rst
190:Godot also provides a way of customizing how everything is drawn inside :ref:`Viewports <class_Viewport>` using “Debug Draw”.

tutorials/scripting/c_sharp/c_sharp_features.rst
99:If you want to connect a signal in the editor, you need to (re)build the project assemblies to see the new signal. This build can be manually triggered by clicking the “Build” button at the top right corner of the editor window.

getting_started/step_by_step/your_first_game.rst
576:Set the ``Playing`` property in the Inspector to “On”.
$ rg "”"
tutorials/export/exporting_pcks.rst
58:and go to Project/Export and click on “Export PCK/Zip”. Also make sure to have
94:validation code as a layer on top. The following example expects a “mod.pck”
96:“mod_scene.tscn” test scene in its root.

tutorials/rendering/viewports.rst
190:Godot also provides a way of customizing how everything is drawn inside :ref:`Viewports <class_Viewport>` using “Debug Draw”.

tutorials/scripting/c_sharp/c_sharp_features.rst
99:If you want to connect a signal in the editor, you need to (re)build the project assemblies to see the new signal. This build can be manually triggered by clicking the “Build” button at the top right corner of the editor window.

getting_started/step_by_step/your_first_game.rst
576:Set the ``Playing`` property in the Inspector to “On”.
```

It's actually a Sphinx **Smart Quotes** feature to convert "plain" quotes `"` and `'` to typographical ones when it builds (and it does it only for text paragraphs and not for inline code or code blocks).

It was disabled recently with #3760 as it creates issues for localization. It could be re-enabled if we agree that all references to Strings with `"some string"` should be made as inline code, and not plain text.

Reply:
I don't know how these characters ended up here. Might've happened while running the content through Antidote, which I tested for grammar checking at the start of this contract.